### DISCUSSION TITLE
Mic input muted randomly and cannot unmute

### DISCUSSION THREAD

**Discussion by digitblues** [jamulussoftware/jamulus#1924] [Technical Support & Help]
I experience a weird situation with my friend connected on Jumulus.  We both running version 3.8.0 on Windows 10 using identical computer hardware.  Interfaces are different.  I use the build-in Realtek and he uses Behringer UMC22.

Our audio was connecting fine for a while.  What happened is out of the blue, after some random people connected to the public server and then left,  he saw the red Muted icon on his mixer screen pop up at the top of his volume slider.  At the same time he could not hear himself on the headphone from Jamulus any more. No matter how many times he tried to click on the Mute button, GRP button and Solo button and turning all of them off, he was not able to get rid of the red Muted icon at the top and not able to hear himself.  He was able to hear me all the time.  We tried disconnecting from the chosen public server and re-connect, the result is the same.  We tried exiting Jamulus and reconnecting, the same result.  Then we both reboot our computers and reconnect, no change, he still cannot hear and red Muted icon stays on.   I had never muted him in all these tests. 

Then when I alone disconnect from the public server, he was suddenly able to hear himself and the red Muted icon was gone.  As soon as I re-connecting to the server, the same thing happened to him again.  I repeat this process several time and the result is the same.

Then I use another computer to run Jamulus and connect to the same public server again.  Everything is now fine on his side with my use of the 2nd computer only.  FYI, the 2nd computer have acquired a different local IP address from the router and that could explain the server is treating my 2nd computer as a new/different user.  

Why is this happening? Is there a way that a total reset can be sent to the server to forget any previous/erroneous settings to start from scratch?

I will be interested in knowing if anyone have similar experience.

Peter

**Comment by ann0see** [comment]
Hi! Sorry to hear that you’re having problems!


Did you check the mute myself button on the bottom left? 

This button (not the button on your mixer strip) evokes the message with the red banner. 

Maybe feedback detection fired wrongly? So it might be a bug?

**Comment by digitblues** [comment]
The Mute myself switch was never engaged.  We also disabled the Feedback
detection just to eliminate all possibilities. No improvement with that
either.

The fact that a new IP address is registered on the public server did not
repeat the same problem must be something to do with the info the server
received on the members.

**Reply by ann0see** [reply]
So it sounds like a bug?
We didn’t hear of anything like that yet, so it sounds suspicious.

Did you try resetting all muted and solo settings (should be in the view menu?)

To answer your question: yes, you can delete the configuration file:

You should be able to find it under
* %APPDATA%\Jamulus\Jamulus.ini 
* ~/.config/Jamulus/Jamulus.ini
* (not sure about macOS)

Do all public/private servers show this issue?
